### PR TITLE
fix: Layout shift on Landing page

### LINKED ISSUE #211: [Bug]: Layout shift on Landing page
### Describe the bug

Clicking the theme selection button on the landing page shifts the header to the right because scroll bar disappears

### To Reproduce

1. Go to http://localhost:3000/
2. Click on the theme selection button
3. Observe the header

### Expected behavior

The header layout will shift right 

### Screenshots/Videos

https://www.loom.com/share/f92818b0718242d68611eb3d01c47eed?sid=8875982d-95d5-4215-8648-47d6b0d0a81e

### Additional context

_No response_

### Please checkmark the following checklist

- [X] I make sure that similar issue is not opened

### PR DESCRIPTION
**Pull request by maheshchandra10**
## What does this PR do?

Fixes #211 

Loom Video: https://www.loom.com/share/6968b11099364a0688e49c0a4d7654b8

## Requirement/Documentation

https://www.radix-ui.com/primitives/docs/components/dropdown-menu

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Are there environment variables that should be set? Just the one which is given in the documentation to run the app locally
- What are the minimal test data to have? Nothing, just the home page
- What is expected (happy path) to have (input and output)? Navbar should not distort while changing the theme 
- Any other important info that could help to test that PR? No

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -100,6 +100,10 @@ body,
   scrollbar-gutter: stable;
 }
 
+nav {
```
Comment (hemantwasthere) [inline styles/globals.css:103]: instead of global class you can add `pl-[calc(100vw-100%)]` in the classname of nav element

**Review by hemantwasthere — CHANGES_REQUESTED** [review]

File: styles/globals.css (hunk 1 above)
Comment (maheshchandra10) [reply styles/globals.css:103]: will do.

Comment (maheshchandra10) [reply styles/globals.css]: @hemantwasthere Request you to please check now.

**Review by hemantwasthere — APPROVED** [review]
LGTM

**Pull request merged**